### PR TITLE
Update docs for upgraded

### PR DESCRIPTION
**Pull request by muhamadazmy**

### CODE REVIEW

**Comment by muhamadazmy** [comment]
@zaibon I am not sure if we should keep the `philosophy` section or not. It seems that's not the case anymore, and was never the case. no?

**Comment by zaibon** [comment]
> I am not sure if we should keep the philosophy section or not. It seems that's not the case anymore, and was never the case. no?

Yes we should keep it (could be that it needs to be change a bit though) and this is still how we want to do things. The 3 mode dev, staging, prod.  Nothing of that changes.

Also can you re-add a diagram. Text is nice but drawing makes things easier to understand.

**Comment by muhamadazmy** [comment]
@zaibon can u pleas take another look

File: docs/upgrade/readme.md [hunk 1]
```diff
@@ -5,103 +5,93 @@
...
 
 Upgrade module depends on network and flist module. This is because is requires network connection to check for new update and the flist module to download the upgrade flist on the node.
 
+
 ## Philosophy
 
 0-OS is meant to be a black box no one can access. While this provide some nice security features it also makes it harder to manage. Specially when it comes to update/upgrade.
 
-Since 0-OS is meant to be driven by some blockchain transaction, the version of the software running on 0-OS could also be driven by a consensus of technical people agreeing on a new version to deploy.
+Hence, zos only trust few sources for upgrade packages. When the node boots up it checks the sources for the latest release and make sure all the local binaries are uptodate before continuing the booting. The flist source must be rock-solid secured, that's another topic for different documentation.
```
Comment (zaibon) [inline docs/upgrade/readme.md]:
```suggestion
Hence, zos only trust few sources for upgrade packages. When the node boots up it checks the sources for the latest release and make sure all the local binaries are up-to-date before continuing the booting. The flist source must be rock-solid secured, that's another topic for different documentation.
```

File: docs/upgrade/readme.md [hunk 2]
```diff
@@ -5,103 +5,93 @@
...
 
-Once a new version of a component is ready, a multisig transaction is created by the 0-OS team and X number of person needs to sign the transaction to approve the deployment of the new version.
+The run mode defines which flist the node is going to use to boot. Run mode can be specified by passing `runmode=<mode>` to the kernel boot params. Currently we have those different run modes.
 
-Using a blockchain as a source for upgrade gives us the ability to create different version of the grid. Version matching the different network of the blockchain:
+- dev: ephemeral network only setup to develop and test new features. Can be created and reset at anytime
+- test: Mostly stable feature that needs to be tested at scale, allow preview and test of new features. Always the latest greatest. This network can be reset sometimes, but should be relatively stable.
```
Comment (zaibon) [inline docs/upgrade/readme.md]:
```suggestion
- test: Mostly stable features that need to be tested at scale, allow preview and test of new features. Always the latest and greatest. This network can be reset sometimes, but should be relatively stable.
```

**Review by zaibon — APPROVED** [review]
LGTM, only saw 2 typos.

**Pull request merged**